### PR TITLE
Update changelog for Mixxx 2.3.6

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,6 +30,13 @@
   [#11709](https://github.com/mixxxdj/mixxx/issues/11709)
 * Controller Preferences: Avoid scrollbars in I/O tabs if Info tab exceeds page height
   [#11756](https://github.com/mixxxdj/mixxx/pull/11756)
+* Broadcast: Improved error message in case of timeout
+  [#11775](https://github.com/mixxxdj/mixxx/pull/11775)
+* Protobuf: Fexed building with v23.4
+  [#11751](https://github.com/mixxxdj/mixxx/pull/11751)
```
Comment (Swiftb0y) [inline CHANGELOG.md:36]:
```suggestion
* Protobuf: Fixed building with v23.4
  [#11751](https://github.com/mixxxdj/mixxx/pull/11751)
```

**Review by Swiftb0y — CHANGES_REQUESTED** [review]

**Comment by daschuer** [comment]
Fixed and integrated latest merge.

The 2.3.6 release and "closing" of the 2.3 branch is a prerequisite for a 2.4 release.

I think we should follow a deadline driven, but "no regression" strategy. 
From that point of view 
Theses Issues are blocking the release:
https://github.com/mixxxdj/mixxx/issues/11545
https://github.com/mixxxdj/mixxx/issues/11785

There are some more PRs targeted the 2.3 branch, nice to have but they can go to 2.4.1 in favor of not stretching the deadline. 

Does that sound reasonable? 

(Will cosponsor that on Zulip)

File: CHANGELOG.md [hunk 2]
```diff
@@ -30,6 +30,15 @@
   [#11709](https://github.com/mixxxdj/mixxx/issues/11709)
 * Controller Preferences: Avoid scrollbars in I/O tabs if Info tab exceeds page height
   [#11756](https://github.com/mixxxdj/mixxx/pull/11756)
+* Broadcast: Improved error message in case of timeout
+  [#11775](https://github.com/mixxxdj/mixxx/pull/11775)
+* Handle setting loop_in and loop_out at the same position
```
Comment (Holzhaus) [inline CHANGELOG.md:35]:
```suggestion
* Handle setting `loop_in` and `loop_out` to the same position
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -30,6 +30,15 @@
...
+  [#11771](https://github.com/mixxxdj/mixxx/pull/11771)
+  [#10600](https://github.com/mixxxdj/mixxx/issues/10600)
+* Fixed build issues with Protobuf v23.4 and with clang 32
```
Comment (Holzhaus) [inline CHANGELOG.md:38]: I think the other points are in present tense
```suggestion
* Fix build issues with Protobuf v23.4 and with clang 32
```

**Comment by daschuer** [comment]
Done.

File: CHANGELOG.md [hunk 4]
```diff
@@ -30,6 +30,24 @@
...
+* Broadcast: Improved error message in case of timeout
+  [#11775](https://github.com/mixxxdj/mixxx/pull/11775)
+* Handle setting `loop_in` and `loop_out` to the same position
+  [#11771](https://github.com/mixxxdj/mixxx/pull/11771)
+  [#10600](https://github.com/mixxxdj/mixxx/issues/10600)
+* Fix build issues with Protobuf v23.4 and with clang 32
+  [#11751](https://github.com/mixxxdj/mixxx/pull/11751)
+  [#11765](https://github.com/mixxxdj/mixxx/pull/11765)
+  [#11762](https://github.com/mixxxdj/mixxx/issues/11762)
+* Disable GL VU-Meters on windows by default. They can be re-enabled via the command line option: `--enableVuMeterGL`.
```
Comment (Holzhaus) [inline CHANGELOG.md:42]:
```suggestion
* Disable GL VU-Meters on Windows by default. They can be re-enabled via the command line option `--enableVuMeterGL`.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -30,6 +30,24 @@
...
+  [#11787](https://github.com/mixxxdj/mixxx/pull/11787)
+  [#11785](https://github.com/mixxxdj/mixxx/issues/11785)
+  [#11789](https://github.com/mixxxdj/mixxx/issues/11789)
+* Library preferences: Uncheck serato metadata export when file metadata export is unchecked
```
Comment (Holzhaus) [inline CHANGELOG.md:46]:
```suggestion
* Library preferences: Uncheck Serato metadata export when file metadata export is unchecked
```

**Comment by daschuer** [comment]
Done + new entry recently merged.

**Review by Holzhaus — APPROVED** [review]
LGTM, thanks.

**Review by Swiftb0y — APPROVED** [review]
LGTM thank you for the PR and for the review

**Pull request merged**